src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,79 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Resources "Create" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+
+        @ignore
+        Scenario: Verify new resource can be created successfully
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  | setting          | value                                          |
+                  | codeValue        | Sick Leave                                     |
+                  | description      | Sick Leave                                     |
+                  | namespace        | uri://ed-fi.org/AbsenceEventCategoryDescriptor |
+                  | shortDescription | Sick Leave                                     |
```

Comment:
Our other files provide the JSON body. Can you change these to JSON bodies instead of a table?